Changelog.md

```
@@ -34,6 +34,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - FEAT: Truncate Tool Calls. Allow extension (cli) #54
 
 ### Fixed
+  - bug/697-can-checkpoint-off-thread (2026-01-24)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)